env-vars.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const dotenv = require('dotenv');
+
+module.exports = async () => {
+  return Object.entries(dotenv.config({ path: '.env' }).parsed).reduce((acc, [key, value]) => {
```

Comment:
```suggestion
  return Object.entries(dotenv.config({ path: '.env' }).parsed  || {}).reduce((acc, [key, value]) => {
```

just to be safe

Reply:
I'm not sure of the implications, looks like trading a runtime error for a silent failure, but probably doesn't matter too much